### ISSUE TITLE
Accessibility:Switch: Red and green color information is not conveyed when switch button highlighted in red are accessed

### ISSUE THREAD

**Issue by puru-hk** [SAP/fundamental-ngx#3496]
Description: Red and green color information is not conveyed when switch button highlighted in red are accessed

Expected: color information about UI should be conveyed to the user

Screenshot: 
![image](https://user-images.githubusercontent.com/32538291/94651310-e7051a80-0315-11eb-9eca-731414b9ee4b.png)

**Comment by puru-hk** [comment]
@Sanjay-1312 Please test this issues

**Comment by Sanjay-1312** [comment]
Description: Red and green color information is not conveyed when switch button highlighted in red are accessed

Expected: color information about UI should be conveyed to the user.

Comment: issue is still reproducible can you please fix the issue.

Screenshot:
![image](https://user-images.githubusercontent.com/78714518/108035091-78bebb80-705c-11eb-9267-da7eda51c74f.png)

**Comment by puru-hk** [comment]
@Betrozov This issues tested by accessibility team. Still it is not fixed. Could you please look into this issue.

**Comment by Betrozov** [comment]
@puru-hk @Sanjay-1312 Please retest this issue

**Comment by Sanjay-1312** [comment]
Hello Colleagues,

The issue is fixed.
Hence closing the ticket.

Thanks and Regards
Sanjay
